frontend/app/src/components/premium/NoPremiumPlaceholder.vue

```diff
@@ -14,7 +13,7 @@ const remoteEmptyScreenLogo =
         class="w-[8rem] h-[8rem] md:w-[16rem] md:h-[16rem] bg-rui-grey-200 p-8 md:p-16 rounded-full mb-8"
       >
         <slot name="logo">
-          <RotkiLogo class="h-full w-full" :url="remoteEmptyScreenLogo" />
+          <RotkiLogo class="h-full w-full" :url="externalLinks.logo.noData" />
```

Comment:
Just a comment here, this is not technically an external link.

It's a dynamic content url, as such it should not be part of the verification process.

In general maybe the dynamic content urls that are not links that the user can click should be part of a different structure instead off external links.